openpype/hosts/houdini/plugins/publish/collect_fbx_type.py

```diff
@@ -0,0 +1,59 @@
+"""Collector for filmboxfbx types.
+
+Collectors act as a pre process for the validation stage.
+It is used mainly to update instance.data
+
+P.S.
+    There are some collectors that run by default
+    for all types.
+
+This plugin is part of publish process guide.
+"""
```

Comment:
```suggestion
"""Collector for filmboxfbx types."""
```

You're basically documenting here how pyblish and openpype work - that's a documentation problem, not one to be put in every collector python file I'd say.